src/main/java/de/ait/secondlife/controllers/ImageContorller.java

```diff
@@ -0,0 +1,131 @@
+package de.ait.secondlife.controllers;
+
+
+import de.ait.secondlife.constants.EntityType;
+import de.ait.secondlife.domain.dto.ImageCreateDto;
+
+import de.ait.secondlife.domain.dto.ImagePathsResponseDto;
+import de.ait.secondlife.domain.dto.ImageRequestDto;
+import de.ait.secondlife.domain.dto.ResponseMessageDto;
+import de.ait.secondlife.services.interfaces.ImageService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.ExampleObject;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.mvc.method.annotation.StreamingResponseBody;
+
+import java.io.InputStream;
+import java.util.Set;
+
+@RestController
+@RequestMapping("/v1/images")
+@RequiredArgsConstructor
+@Tag(name = "Image controller", description = "Controller for some operations with images")
+@ApiResponses(value = {
+        @ApiResponse(responseCode = "400", description = "Bad request", content = @Content(
```

Comment:
can all controller methods return a 400 error? for example getImages()?

Reply:
yes all controller methods can return a 400 error. MaxImageCountException BadFileFormatException FileNameIsWrongException

Reply:
I see all these exceptions in _save_ and _delete_ service methods. 
but what about ImageContorller.getImages()? there is only ImagesNotFoundException - am I wrong?